### PR TITLE
add support to output POM_VERSION, README fix

### LINKED ISSUE #4: "Build the Docker Image" section in README
Should this section be removed? https://github.com/harness-community/drone-maven-docker-image-builder/blob/main/README.md?plain=1#L11-L17

There is no `build_and_push_docker_image.sh` file in the repository.

### PR DESCRIPTION
**Pull request by justAkshitAgrawal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,17 @@
 # Introducing the drone-maven-docker-image-builder Plugin
```
Comment (Ompragash) [inline README.md]:
```suggestion
# Introducing the drone-get-maven-version Plugin
```

File: README.md [hunk 2]
```diff
@@ -1,20 +1,17 @@
...
 
-At Harness, we are dedicated to enhancing Continuous Integration (CI) and Continuous Deployment (CD) processes by providing tools that simplify complex workflows. We understand the significance of seamlessly integrating Maven builds with Docker image creation. That's why we are thrilled to introduce the **drone-maven-docker-image-builder** plugin. This plugin streamlines the process of building and pushing Docker images based on Maven projects, enabling you to effortlessly manage versioning and image creation in your CI/CD pipelines.
+At Harness, we are dedicated to enhancing Continuous Integration (CI) and Continuous Deployment (CD) processes by providing tools that simplify complex workflows. We understand the significance of seamlessly integrating Maven builds with Docker image creation. That's why we are thrilled to introduce the **drone-maven-docker-image-builder** plugin. This plugin streamlines the process of accessing the POM Version, enabling you to effortlessly manage versioning and image creation in your CI/CD pipelines.
```
Comment (Ompragash) [inline README.md]:
```suggestion
At Harness, we are dedicated to enhancing Continuous Integration (CI) and Continuous Deployment (CD) processes by providing tools that simplify complex workflows. We understand the significance of seamlessly integrating Maven builds with Docker image creation. That's why we are thrilled to introduce the **drone-get-maven-version** plugin. This plugin streamlines the process of accessing the POM Version, enabling you to effortlessly manage versioning and image creation in your CI/CD pipelines.
```

File: README.md [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
 
 ### What is the drone-maven-docker-image-builder plugin?
```
Comment (Ompragash) [inline README.md]:
```suggestion
### What is the drone-get-maven-version plugin?
```

File: README.md [hunk 4]
```diff
@@ -1,20 +1,17 @@
...
 
-The **drone-maven-docker-image-builder** plugin is a versatile tool designed to simplify the integration of Maven builds with Docker image creation. This plugin automates the process of building and pushing Docker images by leveraging Maven project information, ensuring a smooth and efficient workflow.
+The **drone-maven-docker-image-builder** plugin is a versatile tool designed to simplify the integration of Maven builds with Docker image creation. This plugin automates the process of fetching the POM version in your maven project.
```
Comment (Ompragash) [inline README.md]:
```suggestion
The **drone-get-maven-version** plugin is a versatile tool designed to simplify the integration of Maven builds . This plugin automates the process of fetching the POM version in your maven project.
```

File: README.md [hunk 5]
```diff
@@ -26,9 +23,6 @@ Integrating the drone-maven-docker-image-builder Plugin into your Harness CI pip
 
     docker run --rm \
     -e PLUGIN_POM_PATH=${POM_PATH} \
-    -e PLUGIN_DOCKER_IMAGE_NAME=${DOCKER_IMAGE_NAME} \
-    -e PLUGIN_DOCKER_USERNAME=${DOCKER_USERNAME} \
-    -e PLUGIN_DOCKER_PAT=${DOCKER_PAT}
     -v $(pwd):$(pwd) \
     -w $(pwd) \
     harnesscommunity/maven-docker-image-builder
```
Comment (Ompragash) [inline README.md]:
```suggestion
    harnesscommunity/get-maven-version
```

File: README.md [hunk 6]
```diff
@@ -44,23 +38,14 @@ In your Harness CI pipeline, you can define the plugin as a step, like this:
             image:  harnesscommunity/drone-maven-docker-image-builder
```
Comment (Ompragash) [inline README.md]:
```suggestion
            image:  harnesscommunity/drone-get-maven-version
```

File: README.md [hunk 7]
```diff
@@ -44,23 +38,14 @@ In your Harness CI pipeline, you can define the plugin as a step, like this:
...
 - **pom_path**: The path to your Maven project. You should replace ${POM_PATH} with the actual path to your Maven project.
 
-- **docker_image_name**: The name of your Docker image. Replace ${DOCKER_IMAGE_NAME} with your desired image name.
-
-- **docker_username**: Your docker username to log into docker hub registry.
-
-- **docker_pat**: Your docker PAT to authenticate the connection.
-
-These environment variables are crucial for configuring and customizing the behavior of the drone-maven-docker-image-builder plugin when executed as a Docker container. They allow you to provide specific values and project information required for building and tagging your Docker image.
+This environment variable is crucial for configuring and customizing the behavior of the drone-maven-docker-image-builder plugin when executed as a Docker container.
```
Comment (Ompragash) [inline README.md]:
```suggestion
This environment variable is crucial for configuring and customizing the behavior of the drone-get-maven-version plugin when executed as a Docker container.
```

File: README.md [hunk 8]
```diff
@@ -44,23 +38,14 @@ In your Harness CI pipeline, you can define the plugin as a step, like this:
...
 
 ### Get Started with the Maven Docker Image Builder Plugin
```
Comment (Ompragash) [inline README.md]:
```suggestion
### Get Started with the Get-Maven-Version Plugin
```

**Review by Ompragash — CHANGES_REQUESTED** [review]

**Comment by justAkshitAgrawal** [comment]
Requested changes done

**Review by Ompragash — APPROVED** [review]

**Pull request merged**